### PR TITLE
Add a github action to automatically push to gcr from HEAD

### LINKED ISSUE #24: Pull code from this repo to run daily experiments

### PR DESCRIPTION
**Pull request by erfanio**
Fixes #24

### CODE REVIEW

File: .github/workflows/push-to-gcr.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    - name: Build the Docker image
+      run: docker build . --file Dockerfile --tag gcr.io/oss-fuzz-base/oss-fuzz-gen:daily
+    - name: Authenticate to gcloud
+      uses: google-github-actions/auth@v2
+      with:
+        credentials_json: '${{ secrets.GCLOUD_CREDENTIAL }}'
+    - name: Set up Cloud SDK
+      uses: google-github-actions/setup-gcloud@v2
+    - name: Push to GCR
+      run: gcloud docker -- push gcr.io/oss-fuzz-base/oss-fuzz-gen:daily
```
Comment (DonggeLiu) [inline .github/workflows/push-to-gcr.yml:24]: @oliverchang Just to double check: Would we prefer continue using this base registry, or the artifact registry?

**Review by DonggeLiu — COMMENTED** [review]
LGTM, thank for addressing this soooo quickly, @erfanio!

File: .github/workflows/push-to-gcr.yml (hunk 1 above)
Comment (DonggeLiu) [reply .github/workflows/push-to-gcr.yml:24]: Artifact registry might be easier for us to accept external images when they request help with experiments.

Comment (erfanio) [reply .github/workflows/push-to-gcr.yml:24]: @DonggeLiu do you want to merge this and come back to this discussion later?

Comment (DonggeLiu) [reply .github/workflows/push-to-gcr.yml:24]: Could we use `us-central1-docker.pkg.dev/oss-fuzz-base/testing/<image-name>`?

Comment (erfanio) [reply .github/workflows/push-to-gcr.yml]: Don

**Review by DonggeLiu — APPROVED** [review]
Thanks!

**Pull request merged**